modules/graphics/waybar.config.nix

```diff
@@ -0,0 +1,458 @@
+                      },
+                      "temperature": {
+                          // "thermal-zone": 2,
+                          // "hwmon-path": "/sys/class/hwmon/hwmon2/temp1_input",
+                          "critical-threshold": 80,
+                          // "format-critical": "{temperatureC}°C {icon}",
+                          "format": "{temperatureC}°C {icon}",
+                          "format-icons": ["", "", ""]
+                      },
+                      "backlight": {
+                          // "device": "acpi_video1",
+                          "format": "{percent}% {icon}",
+                          "format-icons": ["", "", "", "", "", "", "", "", ""]
+                      },
+                      "battery": {
+                          "states": {
+                              // "good": 95,
+                              "warning": 30,
+                              "critical": 15
+                          },
+                          "interval": 5,
+                          "format": "{capacity}% {icon}",
+                          "format-charging": "{capacity}% 󰢟",
+                          "format-plugged": "{capacity}% ",
+                          "format-alt": "{time} {icon}",
+                          // "format-good": "", // An empty format will hide the module
+                          // "format-full": "",
+                          "format-icons": ["󰂎", "󰁺", "󰁻", "󰁼", "󰁽", "󰁾", "󰁿", "󰁿", "󰂀", "󰂁", "󰂂", "󰁹"]
+                      },
+                      "custom/network1": {
+                        "format": "{}",
+                        "interval": 15,
+                        "exec": "${pkgs.wifi-signal-strength}/bin/wifi-signal-strength",
+                        "return-type": "json",
+                        "on-click": "${pkgs.nm-launcher}/bin/nm-launcher",
+                      },
+                      "pulseaudio": {
+                          // "scroll-step": 1, // %, can be a float
+                          "format": "{volume}% {icon} {format_source}",
+                          "format-bluetooth": "{volume}% {icon} {format_source}",
+                          "format-bluetooth-muted": " {icon} {format_source}",
+                          "format-muted": "󰝟 {format_source}",
+                          "format-source": "{volume}% ",
+                          "format-source-muted": "",
+                          "format-icons": {
+                              "headphone": "",
+                              "hands-free": "",
+                              "headset": "",
+                              "phone": "",
+                              "portable": "",
+                              "car": "",
+                              "default": ["", "", ""]
+                          },
+                      },
+        ''
+        + mkLauncherConfigs config.ghaf.graphics.launchers
+        + ''}'';
+
+      # The UNIX file mode bits
```

Comment:
This looks like copy-paste code from somewhere with commented out code between lines 185-192. If it's copy-pasted code, please:
* add attribution link to from where
* check that the source of this CSS has a license that allows use to use it this way

If this is *not* copy-pasted CSS but our own mad CSS development, please accept my apologies.

Reply:
It seems parts of the configuration is from waybar default/example config.
https://github.com/Alexays/Waybar/blob/master/resources/config

Its not full copy paste but some sections contain exactly same commented lines so i agree with Ville it would be good to add attribution to origin.
Probably same for waybar/style.css below

Reply:
Removed the copied parts mostly and references are added both. Also visual glitches are fine tuned.